src/App/i18n.ts

```diff
@@ -3,19 +3,23 @@ import LanguageDetector from "i18next-browser-languagedetector"
 import { initReactI18next } from "react-i18next"
 
 import translationEN from "../../i18n/en"
+import translationIT from "../../i18n/it"
 
 i18n
   .use(LanguageDetector)
   .use(initReactI18next)
   .init({
     debug: process.env.NODE_ENV === "development" ? true : false,
-    fallbackLng: "en",
+    fallbackLng: "it",
```

Comment:
Let's change it back to

```suggestion
    fallbackLng: "en",
```

and add `"it"` to `availableLanguages` in `i18n/index.ts`. Then the language selector will appear in the app settings, too 😉

Reply:
@andywer thank you very much for your review. As you suggested, I've just improved the translations for italian language, adding it into the availableLanguages and I've restored the default language to en.

Reply:
Awesome, thank you so much! 🤩

@ebma Can you give it a try?

Reply:
Sure!

Reply:
thank you so much @andywer . may ask you when do you think to release on app/play store the italian version of Solar wallet (If everything works fine of course) ? Also do you plan in the future to add the Stellar Federation server feature?

Reply:
We don't have a particular date in mind, but I guess it will be in the next few weeks as we've already lined up a bunch of smaller changes waiting to be released.

At the moment we don't plan to roll-out a federation server. We thought about it some while ago, but it hasn't really got any priority right now.